magic_test.py

```diff
@@ -0,0 +1,21 @@
+from tabnanny import check
+from unittest.mock import MagicMock, Mock, ANY
+
+
+def production(a: int = 2):
+    if a%2!=0:
+       return "even"
+    return "odd"
+
+
+def test_productionClass():
+    check = MagicMock(return_value=24)
+    check()
+    val = production(check.return_value)
```

Comment:
This test doesn't make much sense. If you have used a class object and it's method in production method and mocked that object then it would have made more sense.